react/src/mocks/transferAgreements.ts

```diff
@@ -1,3 +1,55 @@
+import {
+  ShipmentState,
+  TransferAgreementState,
+  TransferAgreementType,
+} from "types/generated/graphql";
+import { base1, base2 } from "./bases";
+import { organisation1, organisation2 } from "./organisations";
+
+export const generateMockTransferAgreement = ({
+  type = TransferAgreementType.Bidirectional,
+  state = TransferAgreementState.UnderReview,
+  comment = "Good Comment",
+  isInitiator = true,
+}) => {
+  const iAmSource =
+    (isInitiator && type !== TransferAgreementType.ReceivingFrom) ||
+    (!isInitiator && type === TransferAgreementType.ReceivingFrom);
+
+  return {
+    id: "1",
+    type,
+    state,
+    comment,
+    validFrom: "01-01-2023",
```

Comment:
The BE returns something like `"validFrom": "2023-02-09T17:24:29+00:00"`, or at least when I use the Playground that's the returned date format.
Maybe it's possible that the FE understands multiple formats but it should be assured that understand the format that the BE returns

Reply:
I changed the datetime format in the mocks.